src/tests/ftest/erasurecode/online_rebuild_mdtest.yaml

```diff
@@ -44,19 +44,24 @@ mdtest:
   test_dir: /
   dfs_destroy: True
   manager: MPICH
-  flags: -u
+  flags: "-u -F -C"
   write_bytes: 524288
   read_bytes: 524288
   depth: 10
-  num_of_files_dirs: 10000000
-  stonewall_timer: 10
-  stonewall_statusfile: stoneWallingStatusFile
+  num_of_files_dirs: 100000009
```

Comment:
Why is the last digit 9? It seems random.

Reply:
Must be a mistake... Will fix it.

Reply:
done.